README.md

```diff
@@ -1 +1,44 @@
-# redux-bookstore
\ No newline at end of file
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn start`
```

Comment:
- [ ] Kindly update the readme and ensure it follows [this template](https://github.com/microverseinc/readme-template).